auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/model/Entitlement.java

```diff
@@ -7,6 +7,7 @@
     DATAHUB_DEFAULT_SCALING,
     CDP_AZURE,
     CDP_GCP,
+    CDP_AWS_NATIVE,
```

Comment:
I suggest adding `_CB` after `CDP` to make it clear that it solely influences CB but not experiences. As I see, the UMS change is still unsubmitted, so we can freely rename this now. 😄